### PR TITLE
feat(BoemlyFormControl): Add Slider as InputType

### PR DESCRIPTION
**Pull request by rschiffer**

### CODE REVIEW

File: src/components/BoemlyFormControl/BoemlyFormControl.tsx [hunk 1]
```diff
@@ -120,6 +131,8 @@ export const BoemlyFormControl: React.FC<BoemlyFormControlProps> = ({
         return <DatePicker size={size} {...datePickerProps} />;
       case 'Textarea':
         return <Textarea bgColor="white" {...textareaProps} />;
+      case 'Slider':
+        return <Slider onChange={() => null} ariaLabel="slider" {...sliderProps} />;
```
Comment (lukasbals) [inline src/components/BoemlyFormControl/BoemlyFormControl.tsx:135]: Hmm, I don't like those "default values". But I'm not sure yet what's the best solution for this. Will think about it and comment later ....

**Review by lukasbals — CHANGES_REQUESTED** [review]

File: src/components/BoemlyFormControl/BoemlyFormControl.tsx (hunk 1 above)
Comment (lukasbals) [reply src/components/BoemlyFormControl/BoemlyFormControl.tsx:135]:
```suggestion
      case 'Slider':
        return <Slider onChange={() => undefined} ariaLabel={label || 'slider'} {...sliderProps} />;
```

What about that? I can't find a better solution 🤔

Comment (rschiffer) [reply src/components/BoemlyFormControl/BoemlyFormControl.tsx:135]: Works for me 👍

**Review by lukasbals — APPROVED** [review]
Looks good to me! 🤓

**Pull request merged**